### DISCUSSION TITLE
I have push the code in a repository but github doesn't show it in my cotribution

### DISCUSSION THREAD

**Discussion by UmerGillani36** [community/community#61353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have push the code in a repository but github doesn't show it in my cotribution

**Comment by Aliyan-Ansari** [comment]
go to the right top of the progress table, click on contribution settings and enable private contributions

**Comment by queenofcorgis** [comment]
@UmerGillani36 @Aliyan-Ansari GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
